pkg/pool-manager/pod_pool.go

```diff
@@ -123,7 +124,7 @@ func (p *PoolManager) allocatePodRequestedMac(requestedMac string, pod *corev1.P
 		return err
 	}
 
-	if _, exist := p.macPoolMap[requestedMac]; exist {
+	if isAllocatedToPod, exist := p.macPoolMap[requestedMac]; exist && isAllocatedToPod && !p.allocatedToCurrentPod(podNamespaced(pod), network) {
```

Comment:
why is `isAllocatedToPod` needed? `macPoolMap` is a set with `true` values. I haven't noticed a code assigning false to it. And if there is any, please update documentation of this attribute in its structure

Reply:
done